tests/test_directives.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (c) 2017 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import pytest
+import sys
+
+from flow.directives import Directives, _DirectivesItem
+from flow.directives import NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS
+from flow.directives import WALLTIME, MEMORY, PROCESS_FRACTION
```

Comment:
Group these imports together. I understand you might want the `Directives` class and `no_aggregation` separately, for semantic reasons. But all the directives should be combined.
```suggestion
from flow.directives import (NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS,
                             WALLTIME, MEMORY, PROCESS_FRACTION)
```